performance/collector/es.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import os
+
+import requests
+import six
+
+import datetime
+
+
+from rally import api
+from rally.common import logging
+from rally import exceptions
+from rally.task import exporter
+
+LOG = logging.getLogger(__name__)
+
+
+@exporter.configure(name="es")
+class ESExporter(exporter.Exporter):
+
+    def validate(self):
+        """Validate connection string.
+
+        The format of connection string in ElasticSearch plugin is
+            es:///<path>.<region>
```

Comment:
what will happen in region will be called "west-1.hoolie.net"? I don't think that it will work. 

So I would suggest different format: 

es:///<connection_string>?region=<region_name>

abd then you can use standard process of parsing url and get both connection string that you need and region_name that you need.